community-containers/npmplus/readme.md

```diff
@@ -0,0 +1,17 @@
+## NPMplus
+This container contains a fork of the Nginx Proxy Manager, which is a WebUI for nginx. It will also automatically create a config and cert for AIO.
+
+### Notes
+- Make sure that no other service is using port `80 (tcp)`, `443 (tcp/upd)` or `81 (udp)` on your host as otherwise the containers will fail to start. You can check this with `sudo netstat -tulpn | grep "443\|80\|81"` before installing AIO.
+- After the container was started the first time, please check the logs for errors. Then you can open NPMplus on `https://127.0.0.1:81` (`[::1]` also works) and change the password. 
```

Comment:
Maybe also mention that https://ip.address.of.server:81 should work as well or doesnt it?

Reply:
does not because of NPM_LISTEN_LOCALHOST=true

Reply:
ah I see. Maybe there should be a hint that this will not work then? Also what about situations where you cannot open a browser on the server directly? How the NPM interface then?

Reply:
that is true

Reply:
all right? So now access via ip.address.of.server.81 should work right?

Reply:
yes

Reply:
should we document this?